src/layout/msWorkspace/msWindow.js

```diff
@@ -8,6 +8,8 @@ const { SetAllocation, Allocate } = Me.imports.src.utils.compatibility;
 const WindowUtils = Me.imports.src.utils.windows;
 const { AppPlaceholder } = Me.imports.src.widget.appPlaceholder;
 
+const isWayland = GLib.getenv('XDG_SESSION_TYPE').toLowerCase() === 'wayland';
+
```

Comment:
This is not really needed, but makes sure the "old" code in msWindow.js is run while on XOrg.